src/qibolab/platforms/platform.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+import yaml
+from qibo.config import log, raise_error
+
+from qibolab.platforms.utils import Channel, Qubit
+from qibolab.pulses import Pulse, ReadoutPulse
+
+
+class Platform:
```

Comment:
This is now inheriting from the `AbstractPlatform` as it should. If we stick to the `InstrumentDesign` idea then in principle we could only keep one platform and have different designs. I guess in that case we could consider making the `Platform` a `dataclass`. This may take some time as we would need to refactor the existing qblox driver (using the multiqubit platform) to an `InstrumentDesign`

However, I am not sure if in the end the `InstrumentDesign` approach is necessary at this stage. For example, looking at this platform, we can see that for the most part it is just delegating all methods to the `InstrumentDesign`. Therefore in principle we could completely remove the design and define everything directly at the platform level. Are there any advantages of using the `InstrumentDesign` that I do not see?

Reply:
Moved to #304.